java/api.debugger.jpda/nbproject/project.properties

```diff
@@ -21,6 +21,7 @@ javac.compilerargs=-Xlint:unchecked
 javac.source=1.8
 javadoc.arch=${basedir}/arch.xml
 javadoc.apichanges=${basedir}/apichanges.xml
+bootclasspath.prepend=${build.dir}/disable-release
```

Comment:
I assume this line is intended to disable setting the release flag as the module links against internal classes?

Reply:
Right: 94aebf0